src/routes/Contact/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { Button, Container, Grid, Icon, Label, Link, List } from '../../components';
+import { Image } from 'semantic-ui-react';
+
+const Contact = ({}: {}): any => {
+  return (
+    <React.Fragment>
+      <div className="contact">
+        <iframe
+          src="https://docs.google.com/forms/d/e/1FAIpQLSczVebMK-7iGEuJu2P7Bu88U5zTHrHn4Q2gxkdP83-f3cjPwA/viewform?embedded=true"
+          width="640"
+          height="750"
+          frameBorder="0"
+        >
+          Loading…
+        </iframe>
+      </div>
+    </React.Fragment>
+  );
+};
+
+export default Contact;
```

Comment:
This is great! I'm not familiar with how this form works, would you mind linking some documentation for me or point me to where the contact reports end up?

Reply:
The responses are emailed to the owner, and are also saved in a google sheet here:
https://docs.google.com/spreadsheets/d/1xVLwN8bPslgq-JtmKJKWd_JeIHglklfUeNscE-Bb1YA/edit#gid=1364178731
We can change this to any form, any email and any sheet if needed (We just change the form url). 
I have added your email (so you can test it).
I was thinking of coding it out, but we need backend to store the data, before we do that.